src/shogun/classifier/svm/LibSVM.cpp

```diff
@@ -48,7 +48,7 @@ bool CLibSVM::train_machine(CFeatures* data)
 	struct svm_node* x_space;
 
 	ASSERT(m_labels && m_labels->get_num_labels())
-	ASSERT(m_labels->get_label_type() == LT_BINARY)
+	auto binary_labels = m_labels->as_binary();
```

Comment:
@lisitsyn @vigsterkr this is how labels could be converted in algorithms, if they need an interface to CBinaryLabels for example

Reply:
Looks good.